README.md

```diff
@@ -2,4 +2,4 @@
 
 Postgres backend for [odie](https://github.com/fsmi/odie-client).
 
-Depends on [Flask-restless](https://flask-restless.readthedocs.org/en/latest/). Use `fill_data.py` to create all necessary schemas and tables in your local postgresql instance and to fill it with some sample data.
+Depends on [Flask-restless](https://flask-restless.readthedocs.org/en/latest/) and [Flask-login](https://flask-login.readthedocs.org/en/latest/). Use `fill_data.py` to create all necessary schemas and tables in your local postgresql instance and to fill it with some sample data.
```

Comment:
Missing `Flask-SQLAlchemy`. Should probably just go into `requirements.txt` instead.